### PR TITLE
[refactor] Move some class/functions in test_jit.py to jit_utils.py

### PR DESCRIPTION
**Pull request by jerryzh168**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #26978 [refactor] Remove unimplmented passes
* **#26839 [refactor] Move some class/functions in test_jit.py to jit_utils.py**

Summary:
att

Test Plan:
ci

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

Differential Revision: [D17643010](https://our.internmc.facebook.com/intern/diff/D17643010)

### CODE REVIEW

File: test/jit_utils.py [hunk 1]
```diff
@@ -579,3 +579,27 @@ def wrapper(*args, **kwargs):
                 return fn(*args, **kwargs)
             return wrapper
         return noop_fuser
+
+
+class Observer(torch.nn.Module):
```
Comment (ZolotukhinM) [inline test/jit_utils.py]: I'm not sure about this - it doesn't seem as generic as everything else in this file. It's really a very test specific (even though we have several tests using it). Ideally I think that should live in some other file along with the tests for JIT passes for quantization. If other people think it's fine, I don't mind though.

File: test/jit_utils.py [hunk 2]
```diff
@@ -579,3 +579,27 @@ def wrapper(*args, **kwargs):
...
+    def calculate_qparams(self):
+        return torch.tensor([2.0]), torch.tensor([3])
+
+class WeightObserver(Observer):
+    def __init__(self):
+        super(WeightObserver, self).__init__(torch.qint8)
+
+def get_forward(c):
+    return c._get_method('forward')
+def get_forward_graph(c):
```
Comment (ZolotukhinM) [inline test/jit_utils.py]: This part looks good to me.

File: test/jit_utils.py (hunk 1 above)
Comment (jerryzh168) [reply test/jit_utils.py]: put them in JitTestCase?

**Review by ZolotukhinM — APPROVED** [review]
LGTM!

**Pull request closed without merging**